PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3957,10 +3958,10 @@ bool Notepad_plus::doBlockComment(comment_mode currCommentMode)
 		size_t lineEnd = _pEditView->execute(SCI_GETLINEENDPOSITION, i);
 
 		// empty lines are not commented
-		if (lineIndent == lineEnd)
+		if (lineIndent == lineEnd && !nppGUI._lineCoomentIncBlankLines)

```

Comment:
Made another commit 7e9f359, to make these changes.
However, I am not able to see them on the pull request 2394.
Should I delete this pull request and recreate another pull request?

Regards,